backend/lib/endpoints/schema/posts.js

```diff
@@ -80,6 +80,12 @@ const likeUnlikeCommentSchema = {
     .prop("commentId", S.string().required()),
 };
 
+const sharePostSchema = {
```

Comment:
similarly, this schema might not be necessary if updating the shares is the only purpose, this updating can be done in the socketIO logic